app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :blog
+
```

Comment:
- [ ] **OPTIONAL**
Please consider using the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` in order to call the `update_comment` method every time a new comment has been saved, then make the method private

Reply:
Recommendation implemented. Thank you